giraffe/src/utils/rawFluxDataTable.ts

```diff
@@ -1,21 +1,98 @@
 import Papa from 'papaparse'
+import HtmlEntities from 'he'
+import unraw from 'unraw'
 
 import {parseChunks} from './fluxParsing'
+import {
+  CSV_OBJECT_BASE_NAME,
+  CSV_OBJECT_START_STRING,
+  CSV_OBJECT_END_STRING,
+} from '../constants/rawFluxData'
 
 export interface ParseFilesResult {
   data: string[][]
   maxColumnCount: number
 }
 
-export const parseFiles = (responses: string[]): ParseFilesResult => {
+export const parseFiles = (
```

Comment:
can we make this a new exported function so that we can use the featureflag subsystem in prod to turn it off? creating a "if flag, import NEWparseFiles else import parseFiles" kind of situation? i've just had this problem explode on me too many times.

Reply:
Sure thing, will do.